### DISCUSSION TITLE
Joomla! 3.9.27

### DISCUSSION THREAD

**Discussion by HLeithner** [joomla/joomla-cms#34208] [General]
New Joomla! Installations [.tar.bz2](https://github.com/joomla/joomla-cms/releases/download/3.9.27/Joomla_3.9.27-Stable-Full_Package.tar.bz2) | [.tar.gz](https://github.com/joomla/joomla-cms/releases/download/3.9.27/Joomla_3.9.27-Stable-Full_Package.tar.gz) | [.zip](https://github.com/joomla/joomla-cms/releases/download/3.9.27/Joomla_3.9.27-Stable-Full_Package.zip)
Update from Joomla! 3.9.26 [.tar.bz2](https://github.com/joomla/joomla-cms/releases/download/3.9.27/Joomla_3.9.26_to_3.9.27-Stable-Patch_Package.tar.bz2) | [.tar.gz](https://github.com/joomla/joomla-cms/releases/download/3.9.27/Joomla_3.9.26_to_3.9.27-Stable-Patch_Package.tar.gz) | [.zip](https://github.com/joomla/joomla-cms/releases/download/3.9.27/Joomla_3.9.26_to_3.9.27-Stable-Patch_Package.zip)
Update from Joomla! 3.9.x [.tar.bz2](https://github.com/joomla/joomla-cms/releases/download/3.9.27/Joomla_3.9.x_to_3.9.27-Stable-Patch_Package.tar.bz2) | [.tar.gz](https://github.com/joomla/joomla-cms/releases/download/3.9.27/Joomla_3.9.x_to_3.9.27-Stable-Patch_Package.tar.gz) | [.zip](https://github.com/joomla/joomla-cms/releases/download/3.9.27/Joomla_3.9.x_to_3.9.27-Stable-Patch_Package.zip)
Update from Joomla! 2.5 or previous 3.x releases [.tar.bz2](https://github.com/joomla/joomla-cms/releases/download/3.9.27/Joomla_3.9.27-Stable-Update_Package.tar.bz2) | [.tar.gz](https://github.com/joomla/joomla-cms/releases/download/3.9.27/Joomla_3.9.27-Stable-Update_Package.tar.gz) | [.zip](https://github.com/joomla/joomla-cms/releases/download/3.9.27/Joomla_3.9.27-Stable-Update_Package.zip)

<hr /><em>This discussion was created from the release <a href='https://github.com/joomla/joomla-cms/releases/tag/3.9.27'>Joomla! 3.9.27</a>.</em>

**Comment by HLeithner** [comment]
I noticed after released that I missed a commit before building the final package, since it's only a cosmetic issue it will be fixed in the next regular release.

**Comment by SharkyKZ** [comment]
B/C break in `Joomla\CMS\Helper\MediaHelper`. And the breaking code isn't even used 🤦‍♂️.
